components/post_view/post_body_additional_content/post_body_additional_content.jsx

```diff
@@ -132,7 +132,7 @@ export default class PostBodyAdditionalContent extends React.PureComponent {
     }
 
     isLinkImage(link) {
```

Comment:
Not blocking but it would be great to have test included.  Let me know if you'd like to add one.  (If not, that's totally fine.  I'll just add it at some point.)

Once the test component is setup, you may test the `isLinkImage` function, something similar like so:
```
const testCases = [
    {link: 'http://localhost/some_image1234.jpg', result: true},
    {link: 'http://localhost/some_image1234.JPG', result: true},
    {link: 'http://localhost/not_image', result: false},
    // ... more test cases of links and not
];

testCases.forEach((testCase) => {
    expect(wrapper.instance().isLinkImage(testCase.link).toEqual(testCase.result));
});
```

Reply:
@saturninoabril Awesome, I'll try to add a test! I'm not the most familiar with React but with the example code you posted I should be able to figure it out!

Also @jasonblais I just re-submitted the contribution license agreement. Let me know if it didn't go through again!

Reply:
@Brodan I see it now! Thanks 👍

Reply:
Hey @jasonblais, I'm almost done writing the tests but I can't seem to figure out what to put for the `children` element in the test's `requiredProps`. I've never written any React or any React tests before, but I was able to get most of what I need by looking at other tests, I just can't figure out this last bit. Here's what I have...
```
import React from 'react';
import {shallow} from 'enzyme';

import {FormattedMessage} from 'react-intl';
import PostBodyAdditionalContent from 'components/post_view/post_body_additional_content/post_body_additional_content.jsx';


describe('components/post_view/PostBodyAdditionalContent', () => {
    const post = {
        id: 'post_id_1',
        root_id: 'root_id',
        channel_id: 'channel_id',
        create_at: 1,
    };
    const requiredProps = {
        post,
        children: ********,
        previewCollapsed: '',
        previewEnabled: false,
        isEmbedVisible: true,
        enableLinkPreviews: true,
        hasImageProxy: true,
    };

    test('isLinkImage, should prview image', () => {
        const testCases = [
            {link: 'http://localhost/some_image1234.jpg', result: true},
            {link: 'http://localhost/some_image1234.JPG', result: true},
            {link: 'http://localhost/not_image', result: false},
            // ... more test cases of links and not
        ];
        const wrapper = shallow(<PostBodyAdditionalContent {...requiredProps}/>);
        testCases.forEach((testCase) => {
            expect(wrapper.instance().isLinkImage(testCase.link).toEqual(testCase.result));
        });
    });
});
```
According to the test errors I've been seeing based on what I put in the `********` placeholder, I know it's supposed to be a single ReactElement, but I'm not sure what should be in that element or how to populate that item. Any help?

Reply:
@saturninoabril Tests added! Took me a bit to figure out, but thank you for the help! Hopefully this is ready to get merged now!

Reply:
Thanks!  Just a minor ESLint error
```
/tests/components/post_view/post_body_additional_content/post_body_additional_content.test.jsx

  8:1  error  More than 1 blank line not allowed  no-multiple-empty-lines
```

Please run `make check-style` to see it passed on ESLint 😄

Reply:
@saturninoabril woops, sorry about that! fixed!

Reply:
Cool, thanks!